### DISCUSSION TITLE
logging - symbol separator

### DISCUSSION THREAD

**Discussion by gswifort** [pytest-dev/pytest#13052] [Q&A]
Can I change the separator in logs?

```raw
FAILED tests/test.py::Test::test  → FAILED tests/test.py@Test@test 
```

**Comment by RonnyPfannschmidt** [comment]
Currently that's not supported 

What's the reason for this request

**Reply by gswifort** [reply]
I was looking for a way to quickly find a given test in vscode. However, it turned out that this method works when you enter the file name and then the function after the @, but it does not work when you paste the entire string.
